index.html

```diff
@@ -1772,7 +1773,7 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
                 the correct form for the operation required. op can
                 be assigned one or more interaction verb(s) each
                 representing a semantic intention of an
-                operation.</td><td>optional</td><td>any type (one of <code>readproperty</code>, <code>writeproperty</code>, <code>observeproperty</code>, <code>unobserveproperty</code>, <code>invokeaction</code>, <code>subscribeevent</code>, <code>unsubscribeevent</code>, <code>readallproperties</code>, <code>writeallproperties</code>, <code>readmultipleproperties</code>, or <code>writemultipleproperties</code>)</td></tr></tbody></table><p>Possible values for the <code>contentCoding</code>
+                operation.</td><td>optional</td><td>any type (one of <code>readproperty</code>, <code>writeproperty</code>, <code>observeproperty</code>, <code>unobserveproperty</code>, <code>invokeaction</code>, <code>subscribeevent</code>, <code>unsubscribeevent</code>, <code>readallproperties</code>, <code>writeallproperties</code>, <code>readmultipleproperties</code>, <code>writemultipleproperties</code>, <code>observeAllProperties</code>, or <code>unobserveAllProperties</code>)</td></tr></tbody></table><p>Possible values for the <code>contentCoding</code>
```

Comment:
I think the capitalization in `observeAllProperties` and `unobserveAllProperties` is wrong. There should be **no** capitalization.

Reply:
you are right, however, I have no clue why this happen or where I have to change this. @vcharpenay do you have any idea?

Reply:
lowercase labels are defined in `/td-context/*.jsonld`. That's the part that was missing.

Reply:
thanks